src/index.ts

```diff
@@ -44,6 +47,7 @@ export class Storage<MetaType extends object> implements StorageInterface<MetaTy
     this.logger = logger || defaultLogger
     this.logger.debug({ component: "Storage" }, `Initializing storage (cache size: ${cacheSize}, static schemas: ${schema.length}, static data: ${data.length})`)
 
+    this.schemas = {}
```

Comment:
You should be able to get the schema from `this.ddl`

Reply:
I am able to get each schema by type with `describe` one by one, but not altogether; hence, I figured I would work with the `schema` input instead.

Reply:
not all the schemas are provided statically to storage. In fact, Core provides none.
this.ddl is the schema registry you should be using